scripts/just.config.ts

```diff
@@ -1,12 +1,5 @@
 import type { EsbuildBuildOptions } from "just-scripts";
-import {
-  tscTask,
-  esbuildTask,
-  jestTask,
-  eslintTask,
-  argv,
-  parallel,
-} from "just-scripts";
+import { tscTask, esbuildTask, eslintTask, parallel } from "just-scripts";
```

Comment:
Any opinions on if we need/want a `just-scripts` task for vitest?

Reply:
I don't think we need it. In general I am not the biggest fan of this abstraction layer especially that unless configured, it doesn't allow passing more arguments to a command. I personally find it confusing and it doesn't bring value, sharing config is anyway done through just by extending default config in each package.

However, if someone could show me some clear benefits, I am open to changing my mind

Reply:
Agreed with that, I phrased it as a question to not anchor the comment.